tests/data/test_types.py

```diff
@@ -157,13 +178,41 @@ def test_save_trajectories(self, trajectory_rew, use_chdir, tmpdir):
             chdir_context = contextlib.nullcontext()
             save_dir = tmpdir
 
-        trajs = [trajectory_rew]
+        if not use_rewards:
+            trajs = [_remove_reward(trajectory_rew)]
```

Comment:
I think you can just add `trajectory` to the args list and you'll get a trajectory without reward, and I *believe* `trajectory_rew` will be the equivalent trajectory but with reward added (not that it really matters in this case). So probably no need for this function.